source/Buoy-Tests.package/AssertionCheckerTest.class/instance/testEnforceOneFactFailingRaisingNonDefaultException.st

```diff
@@ -0,0 +1,11 @@
+tests
+testEnforceOneFactFailingRaisingNonDefaultException
+
+	| explanation |
+
+	explanation := 'A false statement'.
+	
+	self
+		should: [ AssertionChecker enforce: [ false ] because: explanation raising: InstanceCreationFailed ]
```

Comment:
The difference between this test and testEnforceOneFactFailingRaisingError is not clear.